workloads/kube-burner-ocp-wrapper/run.sh

```diff
@@ -8,8 +8,10 @@ LOG_LEVEL=${LOG_LEVEL:-info}
 if [ "$KUBE_BURNER_VERSION" = "default" ]; then
     unset KUBE_BURNER_VERSION
 fi
-KUBE_BURNER_VERSION=${KUBE_BURNER_VERSION:-1.6.2}
+KUBE_BURNER_VERSION=${KUBE_BURNER_VERSION:-1.6.3}
 CHURN=${CHURN:-true}
+PPROF=${PPROF:-true}
```

Comment:
Are planning to enable pprof by default for all our workloads?

Reply:
correct.